### ISSUE TITLE
Simple model import error (tensorflow) 

### ISSUE THREAD

**Issue by yashdusing** [Cloud-CV/Fabrik#379]
Importing a simple model of matrix multiplication followed by a sigmoid/softmax from pbtxt gives an error. 
Steps to recreate error
Download & import model from this link (https://www.dropbox.com/s/8y6hwtpzsybrony/SM.pbtxt?dl=0)
Also, bundle.js changes are not being reflected in real time in docker.

**Comment by yashdusing** [comment]
I'd like to take up the import error .

**Comment by Ram81** [comment]
@yashdusing this issue seems to abstract and has a larger scope in changes, please open a new issue with specifics of the model you used.
